envs/monkey_zoo/blackbox/utils/gcp_machine_handlers.py

```diff
@@ -1,35 +1,47 @@
 import logging
+import os
 import subprocess
 
 LOGGER = logging.getLogger(__name__)
 
 
 class GCPHandler(object):
-
+    # gcloud commands
     AUTHENTICATION_COMMAND = "gcloud auth activate-service-account --key-file=%s"
     SET_PROPERTY_PROJECT = "gcloud config set project %s"
     MACHINE_STARTING_COMMAND = "gcloud compute instances start %s --zone=%s"
     MACHINE_STOPPING_COMMAND = "gcloud compute instances stop %s --zone=%s"
 
+    # Default configuration parameters
```

Comment:
Since each of those constants starts with "DEFAULT", I'm not sure what value this comment adds.